app/views/users/show.html.erb

```diff
@@ -1 +1,13 @@
-<h2>This is one post from user!</h2>
\ No newline at end of file
+<%= render partial: 'user', locals: { user: @user}%>
+
+<article class="user_view">
+  <div class="bio_user">
+    <h4>Bio</h4>
+  </div>
```

Comment:
- [ ] To match the provided [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) and as required for the project, kindly make sure you also render the `user's Bio` text as right now it is empty.